src/components/admin.tsx

```diff
@@ -2030,6 +2095,58 @@ function Admin({ user }: adminProps) {
             />
           </Form>
         </Modal>
+        <Modal show={isProfile}>
```

Comment:
I think preferences should be transparent to the conductors so I'm not sure if showing this is necessary. Maybe as pure information but no need to submit anything.

Reply:
Good idea

Reply:
![Screenshot 2023-01-16 at 3 30 39 PM](https://user-images.githubusercontent.com/40650158/212621617-d91e4584-b5f4-4599-b58a-4c197c2f3dbf.png)

A few points here.

1. I'm not sure if I'm missing anything but the UX looks a little untidy here. Could we improve on it ? Maybe we could use a text input instead of plain text or an even table. The goal is to provide some visual context to the user. 
2. For Max Tries, can we add more description so that the conductor knows what "Max Tries" actually means? (the reason why I'm hesitant to displaying this kind of information to users) Else I'll be getting alot of queries to what exactly it will do.
3. Could we hide the home language part if the congregation does not track languages ? I think a null check can do.